server/src/domain/user/dto/update-user.dto.ts

```diff
@@ -49,4 +50,13 @@ export class UpdateUserDto {
   @Optional()
   @IsBoolean()
   memoriesEnabled?: boolean;
+
+  @Optional()
+  @IsString()
+  @IsNotEmpty()
+  avatarColor?: UserAvatarColor;
```

Comment:
This should not be `IsString()` but `IsEnum(UseAvatarColor)`, right? Also can you add the `@ApiProperty()` stuff for named enums, like we do everywhere else?